content/project/contents.lr

```diff
@@ -11,3 +11,11 @@ gutter:
 body: Copy here.
 ---
 last: They can't all be zingers... Check out the BeeWare projects in the `Attic </projects/attic/>`__.
+---
+applications_description: These are applications.
+---
+applications_title: Applications
+---
+utilities_description: These are utilities.
+---
+utilities_title: Utilities
```

Comment:
Seems like these fields should be a description on the child project type, rather than specifically enumerated here.

Reply:
I couldn't figure out how to make that work this entire time, until I was falling asleep last night, and it clicked. It now uses the information from the category page instead of duplicating it in the Project page.